hocr-check

```diff
@@ -57,9 +57,17 @@ def mostly_nonoverlapping(boxes,significant_overlap=0.2):
 ### main
 ################################################################
 
-if len(sys.argv)<1:
+def print_usage():
     print "usage:",sys.argv[0],"[-o] file.html"
+
+if len(sys.argv)>1 and (sys.argv[1] == '-h' or sys.argv[1] == '--help'):
+    print_usage()
+    sys.exit(0)
+
+if len(sys.argv)<1:
```

Comment:
What about using blanks before and after all operators (<, !=, ...)?
That's a style which I personally prefer and which is also quite common.